templates/docker/install-go.sh

```diff
@@ -9,7 +9,7 @@
 
 VERSION="$1"
 
-fname="$(curl -s https://go.dev/dl/ | grep -o "go$VERSION.*.linux-amd64.tar.gz" | head -1)"
+fname="$(curl -s https://go.dev/dl/ | grep -oP "go$VERSION\.\d+\.linux-amd64\.tar\.gz" | head -1)"
```

Comment:
hows this?  Remove that perl grep stuff

```suggestion
fname="$(curl -s https://go.dev/dl/ | grep -oE "go$VERSION.[0-9]+.linux-amd64.tar.gz" | head -1)"
```

Reply:
```bash
⚡ curl -s https://go.dev/dl/ | grep -o "go1.22.*.linux-amd64.tar.gz" | head -1
go1.22.6.linux-amd64.tar.gz">go1.22.6.linux-amd64.tar.gz
py
addyess@addyess-dell /home/addyess/git/charmed-kubernetes/jenkins                                                                                                      main
⚡ curl -s https://go.dev/dl/ | grep -oE "go1.22.[0-9]+.linux-amd64.tar.gz" | head -1
go1.22.6.linux-amd64.tar.gz
py
```

Reply:
the point is to keep it simple. instead of this, perhaps just hardcodd the full go version eg 1.22.6 and avoid smart grep  stuff?